init

```diff
@@ -20,6 +20,11 @@ if [ "$NODE_ROLE" = "it-hybrid" ]; then
   sudo ${SCRIPTDIR}/itv1/util/etcd2-setup.sh $SCRIPTDIR
 fi
 
+# Worker tier - must run the IAM proxy setup before any other containers
+if [ "$NODE_ROLE" = "worker" ] && [ "$VERSION" = "v3" ]; then
```

Comment:
Forgetting to update this when we hit v4 seems like an easy thing to do. Is there an alternative?